packages/core/src/component/component.types.ts

```diff
@@ -22,6 +22,14 @@ export interface ComponentData {
   Variables?: Record<string, unknown>
   Contexts?: Record<string, Record<string, unknown>>
   'URL parameters'?: Record<string, string | null>
+  // eg. source: { path: { docs: null }, query: { embed: everything } }
+  'Route parameters'?: Record<
+    string,
```

Comment:
What do we need the key for? I guess we will always only have 1 set of route parameters when evaluating a formula? If we only ever expect the key to be `source` I think we should be explicit about that in the type